app/views/recipes/index.html.erb

```diff
@@ -0,0 +1,42 @@
+<p style="color: green"><%= notice %></p>
+<%= link_to "New recipe", new_recipe_path %>
+
+<h1>Recipes</h1>
+
+<div id="recipes">
+  <% @recipes.each do |recipe| %>
+
+    <div class="container-solid">
+      
+      <div class="container-column"> 
+        <p>
+          <strong>Recipe:</strong>
+          <%= recipe.name %>
+        </p>
+        <div class="container">
+
+          <% if can? :destroy, recipe %>
+            <%= button_to(
+              "Delete recipe",
+              recipe_path(recipe.id),
+              method: :delete
+            ) %>
+          <%end%>
+
+          <p>
+            <%= link_to "Show more", recipe %>
+          </p>
+        </div>  
+
```

Comment:
- [ ] My suggestion is to ensure that clicking on a recipe item directs you to the recipe details page at `/recipes/:id/`. Therefore, please ensure that clicking on the recipe name also leads to the recipe details page at `/recipes/:id/`. Additionally, please remove the 'show more' link for each recipe on the recipe index page to align with the wireframe provided, as there is no such link on the given wireframe, and make sure that the `Delete recipe` is a link instead of a button.
  
  ### Your recipe index page
  ![image](https://github.com/edahigure/Recipe-app/assets/92303891/ae71d6ed-8bf5-4173-80c7-9432726ca0ad)
  
  ### Wireframe
  ![image](https://github.com/edahigure/Recipe-app/assets/92303891/b6905e73-3347-4810-b3f5-af9788d64d4b)